repo.toml

```diff
@@ -42,14 +42,14 @@ sphinx_conf_py_extra = """
 
   html_domain_indices = False
   html_use_index = False
-  html_extra_path = ["versions1.json"]
+  html_extra_path = ["versions.json"]
```

Comment:
There were two reasons for using `versions1.json`:

- It avoided clobbering the older format that was used before NVIDIA Sphinx Theme.  This way, if someone stumbled on an older version of the docs, the version menu would still work.
- Naming the file `versions1.json` is a bit of a convention across docs projects at NVIDIA, kinda because we've nearly all migrated to using NVIDIA Sphinx Theme.

IMO, not worth changing and kind of worth it to keep as-is for continuity across projects.